docker-for-mac/edge-release-notes.md

```diff
@@ -11,6 +11,26 @@ This page contains information about Docker Desktop Edge releases. Edge releases
 For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.5.1.0
+2020-11-18
+
+> [Download](https://desktop.docker.com/mac/edge/49923/Docker.dmg)
```

Comment:
We normally hide the latest download links, but update the link of the previous release to show the direct link there.

Reply:
Didn’t we stop doing that? The previous release had a direct link already. @usha-mandya what do you think?

Reply:
Ok, fair enough. 👍

Reply:
I'm OK with the direct link. I believe we started adding a direct link after a couple of instances where we forgot to update the previous URL.